.github/workflows/static_analysis.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     - uses: subosito/flutter-action@v2
       with:
         channel: 'stable'
-        flutter-version: '3.13.7'   
```

Comment:
Since the decision is to start from scratch, could we use the last stable version of flutter? 3.24.0

Reply:
It would be amazing and I tried to upgrade to 3.24.0 but it required changes with java and gradle and that's why I went with an incremental step, at least for now